config/config_helpers_test.go

```diff
@@ -539,3 +540,67 @@ func TestGetParentTerragruntDir(t *testing.T) {
 		assert.Equal(t, testCase.expectedPath, actualPath, "For include %v and options %v", testCase.include, testCase.terragruntOptions)
 	}
 }
+
+func TestTerraformBuiltInFunctions(t *testing.T) {
```

Comment:
I just tested a hopefully representative subset of the available functions, I can make it exhaustive if you'd like but I don't think that's really necessary.

Reply:
Agreed, just a few is good enough!